vaadin-radio-group.html

```diff
@@ -0,0 +1,169 @@
+<!--
+@license
+Copyright (c) 2017 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<!--
+`<vaadin-radio-group>` is a Polymer 2 element.
+
+```html
+<vaadin-radio-group>
+</vaadin-radio-group>
+```
+
+@element vaadin-radio-group
+@demo demo/index.html
+-->
+
+<link rel="import" href="vaadin-radio-button.html">
+
+<dom-module id="vaadin-radio-group-default-theme" theme-for="vaadin-radio-group">
+  <template>
+    <style>
+      :host {
+        display: inline-block;
+        outline: none;
+      }
+
+      [part="wrapper"] {
+        display: inline-flex;
+        align-items: center;
+        outline: none;
+      }
+
+      :host([invalid]) [part="wrapper"] {
+        box-shadow: 0 0 2px 2px highlight;
+      }
+    </style>
+  </template>
+</dom-module>
+
+<dom-module id="vaadin-radio-group">
+  <template>
+    <div part="wrapper" id="wrapper">
+      <slot></slot>
+    </div>
+  </template>
+
+  <script>
+    if (!Polymer.Element) {
+      throw new Error(`Unexpected Polymer version ${Polymer.version} is used, expected v2.0.0 or later.`);
+    }
+
+    class GroupRadioElement extends Vaadin.ThemableMixin(Polymer.Element) {
+      static get is() {
+        return 'vaadin-radio-group';
+      }
+
+      static get properties() {
+        return {
+          name: {
+            type: String,
+            reflectToAttribute: true
+          },
+          value: {
+            type: String,
+            notify: true,
+            observer: '_activeValue',
```

Comment:
conventionally we naming observers `_<propertyname>Changed`, in this case it will be `_valueChanged`